x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -16251,6 +16251,7 @@
     "xpack.securitySolution.autocomplete.selectField": "最初にフィールドを選択してください...",
     "xpack.securitySolution.beatFields.errorSearchDescription": "Beatフィールドの取得でエラーが発生しました",
     "xpack.securitySolution.beatFields.failSearchDescription": "Beatフィールドで検索を実行できませんでした",
+    "xpack.securitySolution.callouts.dismissButton": "閉じる",
```

Comment:
We've got a translation team that goes in and fills out the new or updated i18n fields, you just have to delete any unused translations in these files

Reply:
nvm this was just renamed and moved